### PR TITLE
SDQL queries are now on statbus

### PR DESCRIPTION
**Pull request by silicons**
Makes them logged similar to internet sounds
Already in public logs, why not?
I also want to see how many people actually use this.

### CODE REVIEW

**Comment by Cyberboss** [comment]
it only you

**Comment by nfreader** [comment]
>SQL-like queries in SQL
Are we sure this is safe? (I'm assuming it is but I wanna double check)

File: code/modules/admin/verbs/SDQL2/SDQL_2.dm [hunk 1]
```diff
@@ -27,6 +27,7 @@
 	var/list/results = world.SDQL2_query(query_text, key_name_admin(usr), "[usr.ckey]([usr])")
 	for(var/I in 1 to 3)
 		to_chat(usr, results[I])
+	SSblackbox.record_feedback("nested tally", "SDQL query", 1, list("[ckey]", "[query_text]"))
```
Comment (Cyberboss) [inline code/modules/admin/verbs/SDQL2/SDQL_2.dm]: no need to quote the list items

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Comment by Cyberboss** [comment]
It's escaped json data, so its fine

**Pull request merged**